src/org/omegat/Bundle.properties

```diff
@@ -2036,10 +2036,16 @@ GUI_LANGUAGETOOL_URL=URL:
 
 GUI_LANGUAGETOOL_RULES=Rules
 
-GUI_LANGUAGETOOL_BAD_RULE_ID=Rule ID can only contain latin letters and underscores.
+GUI_LANGUAGETOOL_BAD_RULE_ID=Rule ID can only contain Latin letters and underscores.
 
```

Comment:
OK, we can clarify the wording of this string. Can you point me to an example of a rule with a `.` in the ID?